.github/pull_request_template.md

```diff
@@ -1,19 +1,25 @@
-## Done
+### Done
 
```

Comment:
This might just be a personal gripe but I remove the newlines under titles every time. I prefer to have headings and content next to each other and rely on the typography to space them nicely.